### DISCUSSION TITLE
A3 2014 MIGH

### DISCUSSION THREAD

**Discussion by hase77** [Mr-MIBonk/M.I.B._More-Incredible-Bash#288] [General]
Hi,

really thanks for your great work guys. Just patched the MIGH of my 2014 A3 and last last navi cards from 2016/2017 with the Pre-V3.7.1 - MHIG Edition successfully.

First upgraded MIGH SW from MIGH_EU_AU_S1048 to MIGH_EU_AU_K1555. Then installed MIB and run the AIO-Patch. After some horrible fearful minutes with an blank screen at the last reboot upgrade with navi cards 2023/2024 works.

Small hint for the Wiki "hidden menus" key combinations for the A3 2012-2016: The green menu is on the lower left control button + back button combination.

**Comment by harman-f** [comment]
Great!

could you share your backup and LOG files?

**Reply by hase77** [reply]
Just uploaded as MU0415-MHIG_EU_AU_K1555-JTB-07419.02.147777005F.zip

**Comment by enjoey30** [comment]
@hase77 Hey I also got an Audi A3 from 2015 to upgrade, but I’m not sure if my batterie will last it. Did you do the process while the engine we‘re on? I read that it’s not recommended to do it but I wonder what will happen ?

**Reply by hase77** [reply]
I used an Ctek battery charger with the fw update and the patching process to be sure.

**Comment by Siskolamer** [comment]
My car is Q7 whit MHI2 i did that patch on running engine, everting was fine...

**Reply by harman-f** [reply]
This will work most of the time, but you can also get a bad flash from the running engine. 

Just follow the recommendation and use a charger …
